pm_muairss_read/pm_muairss_read.xml

```diff
@@ -0,0 +1,51 @@
+<tool id="pm_muairss_read" name="Cluster outputs (pm-muairs_read)" version="0.1.0" python_template_version="3.5">
+    <requirements>
+        <requirement type="package">pymuon-suite</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        pip show pymuon-suite || ( git clone https://github.com/muon-spectroscopy-computational-project/pymuon-suite /tmp/pymuon-suite && pip install /tmp/pymuon-suite && rm -rf /tmp/pymuon-suite) &&
+        unzip $optimised &&
+        if test -f "params.yaml"; then echo "params.yaml present"; else echo "params.yaml missing" && exit 64; fi 
+        && if test -f "input.cell"; then echo "input.cell present"; else echo "input.cell missing" && exit 64; fi 
+        && out_folder="`python ${__tool_directory__}/get_out_folder.py`" &&
+        pm-muairss -t r input.cell params.yaml && 
+        zip -r out_zip.zip \$out_folder *_clusters* param.yaml input.cell &&
+        find \$out_folder > tree.txt
+    ]]></command>
+    <inputs>
+        <param type="data" name="optimised" format="zip" />
+    </inputs>
+    <outputs>
+        <data label="cluster-folders-$optimised.name" name="out_zip" format="zip" from_work_dir="out_zip.zip"/>
+        <data label="clusters-output-$optimised.name" name="out_cluster" format="txt" from_work_dir="$out_folder/*clusters.txt"/>
+        <data label="clusters-dat-$optimised.name" name="out_cluster_dat" format="txt" from_work_dir="$out_folder/*clusters.dat"/>
```

Comment:
Similarly, I think "dat" can be changed to "data" for a bit more clarity.

Reply:
changed